### ISSUE TITLE
Add attribute to support custom classes for the root editor

### ISSUE THREAD

**Issue by barbu110** [facebook/draft-js#1594]
Hi! I am usually in the situation of having to add multiple draft js instances on one page with different looks and functionalities. A great help with this would be having the possibility to customize the `DraftEditor-root` class name (i.e. to append new classes to it).

What do you think?

**Comment by SKR1N** [comment]
I agree, that would be a great idea, even if you have only one editor on a page. I don't like to use conventionnal plugin names in my style sheets, it feels like it will break at some point with an upgrade.

**Comment by ankitr** [comment]
This would be really useful! I'd be open to making a PR for this.
